### PR TITLE
Fix example in strftime filter plugin help

### PR DESCRIPTION
**Pull request by Klaas-**
##### SUMMARY
The example in the strftime help does not work, I fixed the references.

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION
I am not sure why this lives in the strftime examples and not the to_datetime examples, but for now lets just get the example working

### CODE REVIEW

File: lib/ansible/plugins/filter/strftime.yml [hunk 1]
```diff
@@ -45,9 +45,9 @@ EXAMPLES: |
   vars:
     date1: '2022-11-15T03:23:13.686956868Z'
     date2: '2021-12-15T16:06:24.400087Z'
-    date_short: '{{ date1|regex_replace("([^.]+)(\.\d{6})(\d*)(.+)", "\1\2\4") }}' #shorten microseconds
+    date1_short: '{{ date1|regex_replace("([^.]+)(\.\d{6})(\d*)(.+)", "\1\2\4") }}' #shorten microseconds
```
Comment (Akasurde) [inline lib/ansible/plugins/filter/strftime.yml]: Can you please move this example to https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/filter/to_datetime.yml?

**Comment by Akasurde** [comment]
@Klaas- Hey, are you still working on this?

**Comment by Klaas-** [comment]
> @Klaas- Hey, are you still working on this?

I'll get back to this when I'm back there office. Maybe give me another 1-2 weeks :)

**Comment by Akasurde** [comment]
@Klaas- Thanks for the contribution.

**Pull request merged**